terraform/rancher/project/keycloak.tf

```diff
@@ -0,0 +1,177 @@
+        name: keycloak-credentials
+        key: KC_FOLIO_BE_ADMIN_CLIENT_SECRET
+  - name: KC_HTTPS_KEY_STORE_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_HTTPS_KEY_STORE_PASSWORD
+  - name: KC_HOSTNAME
+    value: ${local.keycloak_url}
+  - name: KC_HOSTNAME_STRICT
+    value: "false"
+  - name: KC_HOSTNAME_STRICT_HTTPS
+    value: "false"
+  - name: KC_HOSTNAME_DEBUG
+    value: "true"
+  - name: KC_DB_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_DB_PASSWORD
+  - name: KC_DB_URL_DATABASE
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_DB_URL_DATABASE
+  - name: KC_DB_URL_HOST
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_DB_URL_HOST
+  - name: KC_DB_URL_PORT
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_DB_URL_PORT
+  - name: KC_DB_USERNAME
+    valueFrom:
+      secretKeyRef:
+        name: keycloak-credentials
+        key: KC_DB_USERNAME
+  - name: KC_PROXY
+    value: edge
+  - name: KC_HTTP_ENABLED
+    value: "true"
+  - name: KC_HTTP_PORT
+    value: "${local.kc_target_http_port}"
+  - name: KC_HEALTH_ENABLED
+    value: "true"
+
+postgresql:
+  enabled: false
+
+externalDatabase:
+  existingSecret: keycloak-credentials
+  existingSecretHostKey: KC_DB_URL_HOST
+  existingSecretPortKey: KC_DB_URL_PORT
+  existingSecretUserKey: KC_DB_USERNAME
+  existingSecretDatabaseKey: KC_DB_URL_DATABASE
+  existingSecretPasswordKey: KC_DB_PASSWORD
+
+logging:
+  output: default
+  level: DEBUG
+
+proxy: edge
+
+enableDefaultInitContainers: false
+
+containerSecurityContext:
+  enabled: false
+
+service:
+  type: NodePort
+  http:
+    enabled: true
+  ports:
+    http: 80
+
+networkPolicy:
+  enabled: false
+
+livenessProbe:
+  enabled: false
+
+customLivenessProbe:
+  httpGet:
+    path: /health/live
+    port: ${local.kc_target_http_port}
+    scheme: HTTP
+  initialDelaySeconds: 0
+  periodSeconds: 1
+  timeoutSeconds: 5
+  failureThreshold: 3
+  successThreshold: 1
+
+readinessProbe:
+  enabled: false
+
+customReadinessProbe:
+  httpGet:
+    path: /health/ready
+    port: ${local.kc_target_http_port}
+    scheme: HTTP
+  initialDelaySeconds: 0
+  periodSeconds: 10
+  timeoutSeconds: 30
+  failureThreshold: 3
+  successThreshold: 1
+
+startupProbe:
+  enabled: false
+
+customStartupProbe:
+  httpGet:
+    path: /health/started
+    port: ${local.kc_target_http_port}
+    scheme: HTTP
+  initialDelaySeconds: 30
+  periodSeconds: 5
+  timeoutSeconds: 1
+  failureThreshold: 60
+  successThreshold: 1
+
+ingress:
+  enabled: true
+  hostname: ${local.keycloak_url}
+  ingressClassName: ""
+  pathType: ImplementationSpecific
+  path: /*
+  annotations:
+    kubernetes.io/ingress.class: alb
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/group.name: folio-tmp.test-2
```

Comment:
I think that this part should be renderd like in minio to point to correct group cluster/ns

Reply:
Thank You.
Forgot to remove debug value for toubleshooting.